recipe/build.sh

```diff
@@ -1,8 +1,15 @@
 #!/bin/bash
 
+source activate "${CONDA_DEFAULT_ENV}"
+
 chmod +x configure
 
-./configure --prefix=$PREFIX --enable-cxx
+if [ "$(uname)" == "Darwin" ];
+then
+    ./configure --prefix=$PREFIX --enable-cxx --build=x86_64-apple-darwin
```

Comment:
Right, it is a little strange to me that this wouldn't work on Mac as I would expect this boils down to different allowed AVX and SSE instructions. Though it seems this is more an optimization for 32-bit machines only. I don't see any mention of 64-bit support. In fact, it looks like Nehalem is the upper end of what they support. See this [thread]( https://groups.google.com/forum/#!topic/mpir-devel/3KCc0_wrEh0 ).